Neos.Flow/Classes/Log/ThrowableStorage/FileStorage.php

```diff
@@ -233,4 +255,35 @@ protected function renderRequestInfo()
 
         return $output;
     }
+
+    /**
+     * Cleans up existing throwable dumps when they are older than the configured
+     * maximum age or the oldest ones exceeding the maximum number of dumps allowed.
+     */
+    protected function cleanupThrowableDumps(): void
+    {
+        if ($this->maximumThrowableDumpAge > 0) {
+            $cutoffTime = time() - $this->maximumThrowableDumpAge;
+
+            /** @var \SplFileInfo $directoryEntry */
+            $iterator = new \DirectoryIterator($this->storagePath);
+            foreach ($iterator as $directoryEntry) {
+                if ($directoryEntry->isFile() && $directoryEntry->getCTime() < $cutoffTime) {
+                    unlink($directoryEntry->getRealPath());
+                }
+            }
+        }
+
+        if ($this->maximumThrowableDumpCount > 0) {
+            // this returns alphabetically ordered, so newest first, as we have a date/time in the name
+            $existingDumps = glob($this->storagePath);
+            if (count($existingDumps) > $this->maximumThrowableDumpCount) {
```

Comment:
`count($existingDumps)` will always return 1 as `$this->storagePath` contains the folder (e.g. `/myFlowPath/Data/Logs/Exceptions`) instead of a glob pattern (e.g. `/myFlowPath/Data/Logs/Exceptions/*`).

Reply:
Oh my… I actually remember writing this in my head:
```php
$existingDumps = glob($this->storagePath . '/*');
```
and then stopping because I wondered about `$this->storagePath` having a trainling slash or not and how `glob` orders things. That got me distracted enough, and I forgot…

Thanks for seeing this! ❤️

Reply:
> I actually remember writing this in my head

A classic. I definitly can relate to that 😄